src/config/configmanager.cpp

```diff
@@ -217,6 +217,9 @@ bool ConfigManager::load() {
 	loadIntConfig(L, CHECK_EXPIRED_MARKET_OFFERS_EACH_MINUTES, "checkExpiredMarketOffersEachMinutes", 60);
 	loadIntConfig(L, COMBAT_CHAIN_DELAY, "combatChainDelay", 50);
 	loadIntConfig(L, COMBAT_CHAIN_TARGETS, "combatChainTargets", 5);
+	loadFloatConfig(L, COMBAT_CHAIN_SKILL_FORMULA_AXE, "combatChainSkillFormulaAxe", 0.9);
+	loadFloatConfig(L, COMBAT_CHAIN_SKILL_FORMULA_CLUB, "combatChainSkillFormulaClub", 0.7);
+	loadFloatConfig(L, COMBAT_CHAIN_SKILL_FORMULA_SWORD, "combatChainSkillFormulaSword", 1.1);
```

Comment:
the configs are separated by type. You added `loadFloatConfig` between `loadIntConfig` calls..

Reply:
Are you sorting by type? I think it's better to follow the same order as the enum (alphabetical), it makes it easier to find and add new modifications.

Reply:
the last modification, it was separated by typo and alphabetical order.